src/client.c

```diff
@@ -584,17 +585,20 @@ static bool decide_startup_pool(PgSocket *client, PktHdr *pkt)
 		if (!ok)
 			break;
 
+		if (!pg_quote_literal(qbuf, val, sizeof(qbuf)))
+			return 0;
```

Comment:
let's only do this when we actually use qbuf (i.e. application_name and varcache ones).

Reply:
Actually only the varcache ones. The appname one we want unquoted.

Reply:
Reason why we should only do this for varcache_set ones, is because we want to support values larger than 128 bytes for others.